packages/tools/nodeGeometryEditor/src/graphSystem/display/inputDisplayManager.ts

```diff
@@ -7,8 +6,14 @@ import styles from "./inputDisplayManager.modules.scss";
 import type { GeometryInputBlock } from "core/Meshes/Node/Blocks/geometryInputBlock";
 import { NodeGeometryBlockConnectionPointTypes } from "core/Meshes/Node/Enums/nodeGeometryConnectionPointTypes";
 import { NodeGeometryContextualSources } from "core/Meshes/Node/Enums/nodeGeometryContextualSources";
+import type { Nullable } from "core/types";
+import type { StateManager } from "shared-ui-components/nodeGraphSystem/stateManager";
+import type { NodeGeometryBlock } from "core/Meshes/Node/nodeGeometryBlock";
 
 export class InputDisplayManager implements IDisplayManager {
+    private _hasHighlights = false;
+    private _predicate = (b: NodeGeometryBlock) => !!(b as any).getExecutionIndex;

```

Comment:
This could be a const directly in the module instead of an instance member as it never changes ?

Reply:
fair!